### ISSUE TITLE
Function Include - creates duplicate lines.

### ISSUE THREAD

**Issue by dlannan-fmad** [unifire-app/aspect#7]
I have a fairly simple file to include. Its a base html, which includes two includes of javascript functions.
I have other includes that work fine (not javascript, css and style twig files) but when I include them, I get a duplication of the first two rows in each inclusion.
See pics.

I have included a really simple main + test1 and test2 file to replicate the issue. 

The duplicated top two lines should not be included in the output.

![image](https://user-images.githubusercontent.com/60128766/90776817-bce53380-e339-11ea-9e83-c9074bf8108a.png)

OS: Debian 10  Linux
Browser: Webkit 
Version: I think its 4.0.37

[tests.zip](https://github.com/unifire-app/aspect/files/5103467/tests.zip)
I load the main.html.twig with the following code:
```
local output, err = aspect:render(fullpath, { })
print(tostring(output))
```

**Comment by bzick** [comment]
eah, it is bug
```
aspect --dump test1-js.html.twig
REFS:{}
VARS:{}
TEMPLATE:
    1| <script>
    2| function do_test_1( something )
    3| {
    4|     something = something + 1
    5| }
    6|
    7| </script>
CODE:
    1| local _self = {
    2| 	v = 1,
    3| 	name = "test1-js.html.twig",
    4| 	blocks = {},
    5| 	macros = {},
    6| }
    7|
    8| function _self.body(__, _context)
    9| 	__:push_state(_self, 1)
   10| 	__.line = 3
   11| 	__([[<script>
   12| function do_test_1( something )
   13| ]])
   14| 	__([[<script>
   15| function do_test_1( something )
   16| {
   17|     something = something + 1
   18| }
   19|
   20| </script>]])
   21| 	__:pop_state()
   22| end
   23|
   24| return _self%

```

strange behavior on a curved parenthesis. I fix it now

**Comment by dlannan-fmad** [comment]
Awesome. Id started debugging the code. Learning how it goes together. Very nice. 
Thanks for the time and effort, your responses are very quick.

**Comment by bzick** [comment]
Fixed in version 2.2
